src/homework9/linkedlist/Node.java

```diff
@@ -0,0 +1,37 @@
+package homework9.linkedlist;
+
+public class Node<T> {
```

Comment:
better make this class as private static nested class

Reply:
Already done.